app/controllers/api/v1/packages_inventories_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Api
+  module V1
+    class PackagesInventoriesController < Api::V1::ApiController
+      load_and_authorize_resource :packages_inventory, parent: false
+
+      resource_description do
+        formats ['json']
+        error 401, "Unauthorized"
+        error 404, "Not Found"
+        error 422, "Validation Error"
+        error 500, "Internal Server Error"
+      end
+
+      api :GET, '/v1/packages_inventories', "List all packages_inventories"
+      def index
+        packages_inventories = @packages_inventories.for_package(params["package_id"])
+        @packages_inventories = packages_inventories.page(page).per(per_page).order('id')
+        render json: { meta: { total_pages: @packages_inventories.total_pages, packages_inventories_count: packages_inventories.size } }.merge(serialized_packages_inventories)
+      end
+
+      private
+
+      def serializer
+        Api::V1::PackageActionsSerializer
```

Comment:
Should we perhaps rename this to `PackagesInventorySerializer` ? cc @steveyken

Reply:
yes, good to keep naming consistent